h/static/scripts/create-group-form.tsx

```diff
@@ -0,0 +1,65 @@
+import { render } from 'preact';
+
+import { Spinner } from '@hypothesis/frontend-shared';
+
+function CreateGroupForm() {
+  return (
+    <>
+      <h1 class="form-header">Create a new private group</h1>
+
+      <form class="form group-form__form">
+        <div class="form-input">
+          <label for="name" class="form-input__label group-form__name-label">
+            Name <span class="form-input__required">*</span>
+          </label>
+          <input
+            type="text"
+            id="name"
+            class="form-input__input group-form__name-input has-label"
+            autofocus=""
+            autocomplete="off"
+            required
+          />
+          <span class="form-input__character-counter is-ready">0/25</span>
+        </div>
+
+        <div class="form-input">
+          <label
+            class="form-input__label group-form__description-label"
+            for="description"
+          >
+            Description
+          </label>
+          <textarea
+            id="description"
+            class="form-input__input group-form__description-input has-label"
+          ></textarea>
+          <span class="form-input__character-counter is-ready">0/250</span>
+        </div>
+
+        <div class="form-actions">
+          <div class="u-stretch"></div>
+
+          <div class="form-actions__buttons">
+            <button class="form-actions__btn btn primary-action-btn group-form__submit-btn">
+              Create group
+            </button>
+          </div>
+        </div>
+      </form>
+
+      <footer class="form-footer">
+        <span class="form-footer__required">
+          <span class="form-footer__symbol">*</span>
+          <span class="form-footer__text">Required</span>
+        </span>
+      </footer>
```

Comment:
I've copied the HTML elements and CSS classes from the legacy create group form.

We're still including the legacy CSS (the `site_css` bundle) because the parts of the legacy page surrounding the new group form are still in use (the Hypothesis logo in the top left, the HTML `<head>` with various things in it, and the `<body>` and some layout/container elements). Therefore, we can just use the legacy CSS classes from our Preact component!

This is a quick way to get a create group form that looks exactly like the legacy one.

It's very close but this doesn't end up looking exactly like the legacy form: the inclusion of `@tailwind base` messes with the layout, changing some of the sizes/margins/paddings just slightly.

Whether we actually want to go with this look in the end or not, I don't know yet.